src/languages/es.ts

```diff
@@ -526,7 +526,7 @@ export default {
         payElsewhere: 'Pagar de otra forma',
         requestAmount: ({amount}: RequestAmountParams) => `solicitar ${amount}`,
         requestedAmount: ({formattedAmount, comment}: RequestedAmountMessageParams) => `solicité ${formattedAmount}${comment ? ` para ${comment}` : ''}`,
-        splitAmount: ({amount}: SplitAmountParams) => `dividir ${amount}`,
+        splitAmount: ({amount, comment}: SplitAmountParams) => `dividir ${amount}${comment ? ` para ${comment}` : ''}`,
```

Comment:
@aimane-chnaif I have modified this copy, but the format is identical to the already existing one, `requestedAmount`.
Should I still request the translation approval here?

Reply:
That's a great catch! Thanks.